schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
+CREATE DATABASE vet_clinic
 
 CREATE TABLE animals (
-    name varchar(100)
+    id              serial PRIMARY KEY,
+    name            varchar(100),
+    date_of_birth   date,
+    escape_attempts int,
+    neutered        boolean,
+    weight_g        decimal
 );
```

Comment:
- [ ] Kindly add a new column called `species` with a string type to the animals table. Remember to add the column by altering the table.